monai/apps/detection/transforms/array.py

```diff
@@ -452,3 +444,102 @@ def __call__(self, boxes_mask: NdarrayOrTensor) -> Tuple[NdarrayOrTensor, Ndarra
             - classification foreground(fg) labels, dtype should be int, sized (N,).
         """
         return convert_mask_to_box(boxes_mask, self.bg_label, self.box_dtype, self.label_dtype)
+
+
+class SpatialCropPadBox(Transform):
```

Comment:
Is it possible to inherit from the `SpatialCrop` transform?
I see much duplicated code.

Thanks.